src/univariate/continuous/pgeneralizedgaussian.jl

```diff
@@ -127,7 +127,7 @@ function rand(rng::AbstractRNG, d::PGeneralizedGaussian)
     g = Gamma(inv(d.p), 1)
 
     # random variable with value -1 or 1 with probability (1/2).
-    b = 2.0 * rand(Bernoulli()) -1
+    b = 2.0 * rand(rng, Bernoulli()) -1
```

Comment:
Usually, `Float64` literals should be avoided. Not sure if it's relevant here but no harm either to follow this guideline:
```suggestion
    b = 2 * rand(rng, Bernoulli()) -1
```

Reply:
Sure, why not. I didn't change anything other than what pertains to the specific issue so far.